doc: nrf91: add quick start to nrf91 GSG (#12900)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Working with nRF91 Series
=========================

> * Added support for Quick Start, a new application in `nRF Connect for Desktop`_ that streamlines the getting started process with the nRF9160 DK.

Reviewer: ```suggestion
* Added support for Quick Start, a new application in `nRF Connect for Desktop`_ that streamlines the getting started process with the nRF91 Series DK.
```
Since Quick start also supports nRF9161 DK.

Question: Placing this under "Working with nRF91 Series" gives an impression that the tool support is from this release (at code level). But we already have the support. Should this be moved to documentation section?

Reviewer 2: I would definitely agree that this is more of a documentation change.

Documentation
=============

> * Updated :ref:`ug_nrf9160_gs` by adding a new section :ref:`nrf9160_gs_installing_software` that replaces the Updating the DK section.

Reviewer 2: ```suggestion
* Updated :ref:`ug_nrf9160_gs` by replacing the Updating the DK section with a new :ref:`nrf9160_gs_installing_software` section.
```
